docs/getting-started/kubernetes.md

```diff
@@ -3,3 +3,65 @@ sidebar_position: 1
 ---
 
 # Install with Kubernetes
+
+## Prerequisites
+
+- This manual uses Helm to deploy Dragonfly on a Kuberenetes cluster. See  [Install Helm](https://helm.sh/docs/intro/install/)
+- A Kuberenetes cluster (See [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) or [Minikube](https://minikube.sigs.k8s.io/docs/start/) if you want to experiment locally)
+- Pick a version frome [here](https://github.com/dragonflydb/dragonfly/pkgs/container/dragonfly%2Fhelm%2Fdragonfly) e.g. `VERSION=v0.15.0`
+
+## Install a standalone master
+
+`helm upgrade --install dragonfly oci://ghcr.io/dragonflydb/dragonfly/helm/dragonfly --version $VERSION`
+
+## Install a standalone master with snapshot taken every minute 
+
+Create or add to your myvals.yaml values file
+
+```
```

Comment:
here is how generic code block looks like
```
storage:
  enabled: true
  requests: 128Mi # Set as desired
```

here is how yaml code block looks like

```yml
storage:
  enabled: true
  requests: 128Mi # Set as desired
```

Reply:
@ashotland  use " ```yml "